src/gerber_files.js

```diff
@@ -15,7 +15,7 @@ function gerberFiles(files, gerberPath) {
     return prev || possibleTypes.indexOf(t) !== possibleTypes.lastIndexOf(t)
   }, false)
   if (!duplicates) {
-    return possibleGerbers
+    return {gerbers: possibleGerbers, types: possibleTypes}
```

Comment:
I can get an array of layer types here; to be used in `processGerbers.js`, but not sure what the best way to pass this data to the jsx build step.

Reply:
You need to count the number of inner copper layers (I think they are called `"icu"` in this version of whats-that-gerber) add two (for top and bottom copper) and then add that to zip-info.json like you are doing already. Or am I missing something?

Reply:
zip-info.json gets written here: https://github.com/kitspace/kitspace/pull/198/files#diff-59fcc8c2f99f544db7e0f3f7fed6b8aaR165

How can I access the types at this point? Should I build the json file earlier?

Reply:
I see, I think this info should be in `stackup` actually.

Reply:
Indeed, it was. Much easier than my initial approach.